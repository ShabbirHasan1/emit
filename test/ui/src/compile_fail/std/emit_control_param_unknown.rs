fn main() {
    emit::emit!(x, "template");
}
