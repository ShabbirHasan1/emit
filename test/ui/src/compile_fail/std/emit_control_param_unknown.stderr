error: unknown argument `x`; available arguments are `mdl`, `extent`, `props`, `rt`, `when`, `evt`
 --> src/compile_fail/std/emit_control_param_unknown.rs:2:17
  |
2 |     emit::emit!(x, "template");
  |                 ^
